cygwin.rst

```diff
@@ -0,0 +1,194 @@
+*****************************
+Docs in Windows (Cygwin) 
+*****************************
+
+.. _cygwin-docs:
+
+Cygwin
+-------------------------------
+Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows 
+command propmpt.
+
+Install `cygwin <https://cygwin.com/install.html/>`_ and add its path into windows environmental variable.
+This will allow you to `use cygwin from cmd promopt <https://www.howtogeek.com/howto/41382/how-to-use-linux-commands-in-windows-with-cygwin/>`_.
+
+
+.. _python:
+
+Python 3
+-------------------------------
+You need to install `python 3 <https://www.python.org/downloads/>`_.
+During the installation, you can select to set up pip tool as well as add python path to the Path environment variable.
+
+
+
+.. _virtualenv:
+
+Virtual Environment
+-------------------------------
+Virtual environment is a useful tool particularly if you work with multiple projects, each requires different dependencies and python versions. By using virtualenv, you can create seperated virtual environment and install packages as you need without affecting the main 
+python installation.
+
+To install virtualenv in windows, you can basically use pip command, which was already installed with python 3.6.2.
+
+.. code-block:: rest
+
+  $ pip install virtualenv
+
+.. note::
+  You need to make sure you have already installed pip in the previous step.
+
+Create a new directory for your odkdocs work:
+
+.. code-block:: rest
+
+  $ mkdir odk
+    
+You have two options: 
+
+  - Use the native virtualenv.
+  - Use virtualenvwrapper.
+
+
+.. _native-virenv:
+
+Native virtualenv
+~~~~~~~~~~~~~~~~~~~~~
+Create a new python 3 virtual environment, "odkenv" is the name of the virtualenv, you are free to add your own.
+
+.. code-block:: rest
+
+  $ virtualenv --python=<python path/python.exe> odkenv
+ 
+After creating python3 virtualenv in the previous step, multiple files are coppied into the folder odkenv.
+
+.. code-block:: rest
+
+  $ ls odkenv
+
+The folder Scripts contains all virtualenv control as bat files.
+
+To activate the odkenv:
+
+.. code-block:: rest
+
+  $ cd odkenv
+  .
+  .
+  .
+  $ cd Scripts
+
+  $ odk/odkenv/Scripts/activate.bat
+
+
+To deactivate the odkenv:
+
+.. code-block:: rest
+
+  $ odk/odkenv/Scripts/deactivate.bat
```

Comment:
does `$ deactivate` not work in Windows?

Reply:
If you use only virtualenv, you can run deactiavte.bat after you cd into the file.
You can run just deactivate if you install virtualenvwrapper for windows.

Reply:
interesting. On Mac/Linux those commands are part of the base venv package.
So it sounds like the wrapper provides useful tools and we should suggest its use for Windows people, as you have done.